server/api/dataset.py

```diff
@@ -58,6 +58,9 @@ def __init__(self):
         self.route('POST', (), self.createDataset)
         self.route('POST', (':id', 'image'), self.addImage)
         self.route('POST', (':id', 'zip'), self.addZipBatch)
+        self.route('POST', (':id', 'zipS3'), self.initiateZipUploadToS3)
```

Comment:
Can we can rename this to `POST /dataset/:id/zip` and rename the other 2 as well.

Then rename the legacy endpoint to `POST /dataset/:id/zipFile` (or something similar) as a temporary measure to keep it working and get it out of the way. We'll just need to update front-end references to this as well.